# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libnetmd/pid.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -u unstable/libnetmd.c unstable-new/libnetmd.c
--- unstable/libnetmd.c	2004-03-29 22:47:01.000000000 +0200
+++ unstable-new/libnetmd.c	2006-07-02 22:01:50.000000000 +0200
@@ -44,6 +44,14 @@
 	{0x54c, 0xc6}, /* Sony MZ-N10 */
 	{0x54c, 0xc8}, /* Sony MZ-N710/N810 */
 	{0x54c, 0xc9}, /* Sony MZ-N510/N610 */
+	{0x54c, 0x188}, // MZ-N920
+	{0x54c, 0x18a}, // LAM-3
+	{0x54c, 0x1e9}, // MZ-DH10P
+	{0x54c, 0x219}, // MZ-RH10
+	{0x54c, 0x21B}, // MZ-RH910
+	{0x54c, 0x21d}, // CMT-AH10
+	{0x54c, 0x286}, /* Sony MZ/RH-1 */
+	{0x54c, 0x287}, /* Sony MZ/RH-1 HiMD mode*/
 	{0, 0} /* terminating pair */
 };
 
